sip/headers.go

```diff
@@ -920,6 +920,7 @@ func (h *ContentTypeHeader) headerClone() Header { return h }
 // RouteHeader  is Route header representation.
 type RouteHeader struct {
 	Address Uri
+	Next    *RouteHeader
```

Comment:
Sorry but I have removed this linked list parsing for comma seperated values. 
They add more complexity, so I am not sure have you rebased on latest?

I would like to add fix for BYE

Reply:
Thanks, fixed that now.